src/main/js/helpers/region.js

```diff
@@ -208,14 +208,51 @@ const shouldHideAllRegions = (regionList, graphTargets) =>
  * @returns {boolean} true if displayed targets is equal to 1, false otherwise
  */
 const isSingleTargetDisplayed = (graphTargets) => graphTargets.length === 1;
+/**
+ * Check all the regions within a graph is same or not
```

Comment:
You are doing more than checking all the regions in this function, you are hiding them as well. Instead you could do just the checking here and just call hideAll if you decide to hide. 
They are shown by default anyway, so you dont need to do, aria-hidden -> false for all the regions, if I am not wrong.

Reply:
no they are not shown by default i tried to do so but it did not worked.

Reply:
"they are not shown by default" Wait what? Where is it hidden by default?

Reply:
it's working it's showing by default you were right

Reply:
but we set , aria-hidden -> false for all the regions because when we click on legend we are calling the areRegionSame() so when all legends are clicked (config.showTarget.length becomes 0) and all regions are hidden so to show again we are call the function areRegionSame() after hiding we have to show again that is why we are setting the aria-hidden -> false.

Reply:
Yeah I am not understanding this at all. Is there a unit test scenario for this?

Reply:
now i am handling that case in the processRegion only if areRegionSame is true then show region only for legend click. 

Yes for line we have toggleRegion if (showTarget === 1) but for pairedResult we don't have toggle for the condition (showTarget === 1) in that case we are checking (graphContext.content === 1).